libwild/src/timing.rs

```diff
@@ -4,18 +4,21 @@ use crate::args::CounterKind;
 use crate::error::AlreadyInitialised;
 use crate::error::Result;
 use crate::perf::CounterList;
-use anyhow::Context;
 use anyhow::anyhow;
 use crossbeam_queue::ArrayQueue;
 use std::fmt::Display;
 use std::path::PathBuf;
+#[cfg(feature = "perfetto")]
+use anyhow::Context;
+#[cfg(feature = "perfetto")]
 use std::sync::Mutex;
 use std::time::Duration;
 use std::time::Instant;
 use tracing::field::Visit;
 
 const PERFETTO_ENV_VAR: &str = "WILD_PERFETTO_OUT";
 
+#[cfg(feature = "perfetto")]
```

Comment:
I just published version 0.3.0 of perfetto-recorder, which adds support for Windows. At least it builds and the tests pass on Windows.